packages/tools/generator-pie-component/src/app/index.ts

```diff
@@ -45,11 +51,82 @@ export default class extends Generator {
             undefined,
             { processDestinationPath },
         );
+
+        // Update YAML files
+        this._addPercyTokenEnvVar();
+        this._addGithubLabel();
+    }
+
+    _readAndParseYaml (filePath:string): Document {
+        const yamlSrc = this.fs.read(filePath);
+        return YAML.parseDocument(yamlSrc);
+    }
+
+    /**
+     * In this method we intentionally bypass Yeomans standard behaviour of asking confirmation for every file update
+     */
+    _stringifyYamlAndWriteFile (filePath:string, yamlDoc:Document) {
+        const yamlStr = yamlDoc.toString({ lineWidth: 0 });
+        fs.writeFileSync(filePath, yamlStr, { encoding: 'utf-8' });
+    }
+
+    /**
+     * Add placeholder Percy token environment variable to ci.yml
+     */
+    _addPercyTokenEnvVar () {
+        const { percyComponentName } = this.props;
+        const yamlFilePath = this.destinationPath('.github/workflows/ci.yml');
+
+        // Read file
+        const yamlDoc:any = this._readAndParseYaml(yamlFilePath);
```

Comment:
Can `any` be replaced by the `Document` type from the `yaml` package?

Reply:
Actually I've noticed you're already using that type above so this should be an easy fix 👍🏻